app/main.py

```diff
@@ -1 +1,47 @@
-# write your code here
+class SoftwareEngineer:
+    def __init__(self, name: str) -> None:
+        self.name = name
+        self.skills = []
+
+    def learn_skill(self, skill: str) -> None:
+        self.skills.append(skill)
+
+
+class FrontendDeveloper(SoftwareEngineer):
+    def __init__(self, name: str) -> None:
+        super().__init__(name)
+        self.skills += ["JavaScript", "HTML", "CSS"]
```

Comment:
we don`t need use [ ] here, in my opinion

u can write: self.skills += "JavaScript", "HTML", "CSS",
and it`s still works

Reply:
As for me, you need it here
It won't confuse anybody and will be more readable